examples/gadgets/withfilter/trace/network/README.md

```diff
@@ -0,0 +1,127 @@
+# trace network example (with filter)
+
+This example shows how to use different packages to create an application to
+trace network connections filtering by containers. Beyond that, network events
+are enriched with Kubernetes IP addresses and Kubernetes destinations, e.g. pod
+or service names.
+
+## How to build
+
+```bash
+$ go build .
+```
+
+## How to run in a terminal
+
+``` console
+$ sudo ./network
```

Comment:
nit: we use bash for console snippets.

Reply:
I've replaced console with bash in README.md